crates/api_crud/src/user/create.rs

```diff
@@ -476,35 +474,32 @@ async fn create_local_user(
   Ok(inserted_local_user)
 }
 
+/// Returns true if email was sent.
 async fn send_verification_email_if_required(
   context: &Data<LemmyContext>,
   local_site: &LocalSite,
   local_user: &LocalUser,
   person: &Person,
 ) -> LemmyResult<bool> {
-  let mut sent = false;
-  if !local_user.admin && local_site.require_email_verification && !local_user.email_verified {
-    let local_user_view = LocalUserView {
-      local_user: local_user.clone(),
-      local_user_vote_display_mode: LocalUserVoteDisplayMode::default(),
-      person: person.clone(),
-      counts: PersonAggregates::default(),
-    };
```

Comment:
Building this fake local_user_view shouldn't have been necessary, and is potentially unsafe, so most of this PR is boilerplate to change function arguments to `local_user`